src/Compatibility/Microsoft.DotNet.PackageValidation/build/Microsoft.DotNet.PackageValidation.targets

```
@@ -77,6 +77,19 @@
       </_PackageValidationIntermediateBaselineFileContent>
     </PropertyGroup>
 
+    <PropertyGroup Condition="'$(PackageValidationBaselineVersion)' == ''">
+      <_PackageValidationIntermediateBaselineFile>$(MSBuildProjectExtensionsPath)$(MSBuildProjectFile).packagevalidation.g.targets</_PackageValidationIntermediateBaselineFile>
+      <_PackageValidationIntermediateBaselineFileContent>
+<![CDATA[<Project>
+  <Target Name="ResetEnablePackageValidation" BeforeTargets="RunPackageValidation">
+    <PropertyGroup>
+     <EnablePackageBaselineValidation>false</EnablePackageBaselineValidation>
```

nit: wrong indentation

```suggestion
      <EnablePackageBaselineValidation>false</EnablePackageBaselineValidation>
```